### ISSUE TITLE
meson: libatalk.so.18 error on Debian

### ISSUE THREAD

**Issue by rdmark** [Netatalk/netatalk#968]
While the issue was fixed on NetBSD, now Debian seems affected:

```
/usr/local/sbin/netatalk: error while loading shared libraries: libatalk.so.18: cannot open shared object file: No such file or directory
```

https://github.com/Netatalk/netatalk/actions/runs/9046362746/job/24857111621?pr=959

Autotools builds are not affected.

**Comment by rdmark** [comment]
Frustratingly, this does not reproduce on a real Debian 12 system in my attempts. Either systemd or sysv init styles...

**Comment by rdmark** [comment]
I disabled the start/stop steps in the github CI job that was triggering this issue. This is a reminder to restore those steps once the underlying issue has been addressed.

**Comment by dgsga** [comment]
Working on it, the root cause is how /etc/ld.conf varies from distro to distro. If I set install_rpath for all executables to libdir then all is well. The only two exceptions to this rule are Omnios and NetBSD so I'll write workarounds for these.

**Comment by dgsga** [comment]
Finally fixed in PR #975
